File: mall-manager-web/src/main/resources/resources/js/xmall/controller/sellerController.js
xmall.controller('sellerController', function ($scope, $controller, sellerService) {
    $controller('baseController', {$scope: $scope});

    $scope.searchEntity = {};
    $scope.sellerEntity = {};

    /**
     * 获取商家列表
     * @param pageNum 页码
     * @param pageSize 每页条数
     */
    $scope.list = function (pageNum, pageSize) {
        sellerService.list(pageNum, pageSize, $scope.searchEntity).success(function (response) {
            $scope.sellerList = response.list;
            $scope.paginationConf.totalItems = response.total;
        })
    };

    /**
     * 获取商家详情
     * @param sellerId 商家id
     */
    $scope.findById = function (sellerId) {
        sellerService.findById(sellerId).success(function (response) {
            $scope.sellerEntity = response;
        })
    };

    /**
     * 商家状态设置
     * @param sellerId 商家ids
     * @param status 要设置的状态
     */
    $scope.updateStatus = function (sellerId, status) {
        sellerService.updateStatus(sellerId, status).success(function (response) {
            if (response.code === 200) {
                $scope.reloadList();
            } else {
                alert(response.message)
            }
        });
    }

});	

File: mall-manager-web/src/main/java/life/eks/xmall/controller/seller/SellerController.java
package life.eks.xmall.controller.seller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.github.pagehelper.PageSerializable;
import life.eks.xmall.api.seller.SellerService;
import life.eks.xmall.common.web.Response;
import life.eks.xmall.pojo.Seller;
import org.springframework.web.bind.annotation.*;

/**
 * @author Eks OU
 */
@RestController
@RequestMapping("/seller")
public class SellerController {

    @Reference(version = "1.0.0")
    private SellerService sellerService;

    @PostMapping("/list")
    public PageSerializable<Seller> list(@RequestParam(defaultValue = "1") int pageNum,
                                         @RequestParam(defaultValue = "10") int pageSize,
                                         @RequestBody Seller seller) {
        return sellerService.list(seller, pageNum, pageSize);
    }

    @GetMapping("findById")
    public Seller findById(String id) {
        return sellerService.findById(id);
    }

    @PostMapping("/updateStatus")
    public Response updateStatus(String id, String status) {
        try {
            sellerService.updateStatus(id, status);
            return Response.success();
        } catch (Exception e) {
            return Response.fail();
        }
    }
}
